backend/migrations/versions/d1d6da1b2d67_add_columns_worksheet_query_result_model.py

```diff
@@ -0,0 +1,44 @@
+"""add_columns_worksheet_query_result_model
+
+Revision ID: d1d6da1b2d67
+Revises: d274e756f0ae
+Create Date: 2024-09-10 14:34:31.492186
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd1d6da1b2d67'
+down_revision = 'f87aecc36d39'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column('worksheet_query_result', 'status', nullable=True)
```

Comment:
Don't we need to add the `worksheetQueryResultUri` column? or was it already created?

Reply:
Column is added, so silly of me.